src/main/target/SPRACINGF3NEO/config.c

```diff
@@ -49,13 +49,20 @@
 #include "fc/config.h"
 
 #ifdef USE_TARGET_CONFIG
+
+#include "config_helper.h"
+
+static targetSerialPortFunction_t targetSerialPortFunction[] = {
+    { 0,              FUNCTION_MSP },
+    { TELEMETRY_UART, TELEMETRY_PROVIDER_DEFAULT },
```

Comment:
While we are at it, can we have the defines for `TELEMETRY_UART` and `GPS_UART` moved into `config.c`? Having to thread around these 'inofficial' defines in `target.h` makes target updates harder for no good reason.

Reply:
Good thinking.